data/vocab/relation.ttl

```diff
@@ -34,12 +36,14 @@
 # We encourage users to provide feedback and suggestions to continually improve and update this vocabulary.
 # For versioning, updates, and more detailed documentation, please visit [https://bibfra.me/?].
 
-# ------------------------------
-# Created by: EBSCO Information Systems
-# Version: 1.0
-# Created: July 30, 2024
-# Updated: July 30, 2024
-# ------------------------------
+rel: a rdfs:Vocabulary ;
+    rdfs:label "Builde Relation" ;
```

Comment:
```suggestion
    rdfs:label "Relation" ;
```

Unless you want this to display as "Builde Relation" on the sidebar in the documentation website.